gulpfile.js

```diff
@@ -1075,14 +1075,14 @@ function createCesiumJs() {
         var moduleId = file;
         moduleId = filePathToModuleId(moduleId);
 
-        var assignmentName = "['" + path.basename(file, path.extname(file)) + "']";
+        var assignmentName = '[\'' + path.basename(file, path.extname(file)) + '\']';
         if (moduleId.indexOf('Shaders/') === 0) {
             assignmentName = '._shaders' + assignmentName;
         }
 
         var parameterName = moduleId.replace(nonIdentifierRegexp, '_');
 
-        moduleIds.push("'./" + moduleId + "'");
+        moduleIds.push('\'./' + moduleId + '\'');
```

Comment:
I wish there was a rule for "allow double quotes when the quoted string contains at least one single quote..."